src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/ElytraCap.java

```diff
@@ -0,0 +1,25 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.armor;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.DamageableItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+
+/**
+ * The {@link ElytraCap} negates damage taken when crashing into a wall using elytra.
+ *
+ * @author Seggan
+ */
+public class ElytraCap extends SlimefunArmorPiece implements DamageableItem {
+
+    public ElytraCap(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, PotionEffect[] effects) {
```

Comment:
Also since this isn't gonna have potion effects you could remove the last parameter here, why does this extend `SlimefunArmorPiece` anyway?

Reply:
@LinoxGH told me to

Reply:
also i have plans to turn this into an armor set